driver-core/src/main/com/mongodb/ReadPreferenceHedgeOptions.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb;
+
+import com.mongodb.annotations.Immutable;
+import org.bson.BsonBoolean;
+import org.bson.BsonDocument;
+
+/**
+ * Options to apply to hedged reads in the server.
+ *
+ * @since 4.1
+ * @mongodb.server.release 4.4
+ */
+@Immutable
+public final class ReadPreferenceHedgeOptions {
+    private final boolean hedgedReads;
+
+    /**
+     * Gets whether hedged reads are enabled in the server.
+     *
+     * @return true if hedged reads are enabled in the server
+     */
+    public boolean getHedgedReads() {
```

Comment:
Nit: why not `isEnabled`?  That would match the field name in the document, and seems appropriate as a simpler name in a class that already has `hedge` in its name.

Reply:
I kept the old variable name when I did not have an Options class. I have changed it now.